.env

```diff
@@ -0,0 +1,2 @@
+VITE_DATA_VERSION="1.0.0"
```

Comment:
I _don't think_ we have the concept of major/minor/patch in the UI/Data versioning. I presume just an integer counter would be sufficient.  At least for now..